.github/workflows/tf_apply_production.yml

```diff
@@ -17,6 +17,7 @@ env:
   TF_VAR_log_analytics_workspace_id: ${{ secrets.LOG_ANALYTICS_WORKSPACE_ID }}
   TF_VAR_log_analytics_workspace_key: ${{ secrets.LOG_ANALYTICS_WORKSPACE_KEY }}
   TF_VAR_slack_webhook_url: ${{ secrets.SLACK_WEBHOOK_URL }}
+  TF_VAR_notify_test_api_key: ${{ secrets.NOTIFY_TEST_API_KEY }}
```

Comment:
It might be more of a `NOTIFY_DOC_API_KEY` and it would be okay to store in plain view I assume as this is only an example and not a real key. CC @whabanks for confirmation.

Reply:
Thanks @jimleroyer! The other reason why I put it in a secret is because if I put the API key in plain text, then it will trigger the alarm again 😅  since github will detect is a leaked notify key (it only looks at the format). By putting it in a secret, it bypasses this even though the key is not that secret since it is already in the documentation. There could be a better way of handling this though, just not sure what it could be.

Reply:
It makes sense to put it in the secrets to not trigger alarms again. To Jimmy's point on the naming, it is more of a `DOC` or `EXAMPLE` key than a `TEST` key. Test implies that the key could be used somewhere.

Reply:
Thanks all! I have changed it to be NOTIFY_DOC_API_KEY